docs/guide.mdx

```diff
@@ -0,0 +1,419 @@
+---
+sidebar_position: 10
+sidebar_label: "Walkthrough"
+id: deployment-guide
+title: Deployment Guide
+description: Deployment Guide
+slug: /walkthrough
+---
+
+This guide provides detailed instructions for deploying a Kwil network and for interacting with the nodes in the network.
+
+## Deploy A Standalone Kwil Node
```

Comment:
Should it be "Deploy A Standalone Kwil Node" or "Deploy a Standalone Kwil Node" ?

Reply:
Lower case articles and prepositions.